github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

```diff
@@ -500,6 +504,14 @@ orgs:
         privacy: secret
         repos:
           kubesecondarydns: admin
+      ipam-extensions:
+        description: ""
+        maintainers:
+          - maiqueb
+          - qinqon
+        privacy: secret
+        repos:
+          ipam-extensions: admin
```

Comment:
nit: as stated on the mailing list, we should cut down on repo access as far as possible.

Reply:
what change is required then ? to keep just maiqueb ?
at least maiqueb should have write access please (for example for tags)

at least for now, we can iron the details bit later as agreed please

Note please that we would still need to be able to create tags / merge (unless tide takes care of that)
and to make sure github internal branch protection is enabled etc